sql-cli/sql_cli/sql_directory_parser.py

```diff
@@ -121,7 +139,7 @@ def get_sql_files(directory: Path, target_directory: Path) -> set[SqlFile]:
     :returns: the sql files found in the directory.
     """
     return {
-        SqlFile(root_directory=directory, path=child, target_directory=target_directory)
+        SqlFile(root_directory=directory, path=child, target_directory=target_directory)  # type: ignore
```

Comment:
I probably know why you added `# type: ignore`, I guess it is to coupled with `target_directory` being set. But before merging we should decouple this in `SqlFile`, or?

Reply:
Could you revert this please since you know pass a target_directory.